### PR TITLE
Use Yarn Workspaces

### PR DESCRIPTION
**Pull request by followyouhome**
### What was a problem?

We have two projects with Recomponents as dependency. In case of some bugs / breaking change test latest build of Recomponents takes lot of manual actions to replace build in node_modules on each update in this project.
By default symlink doesn't work with Webpack and Vue CLI because of symlink issue.

### How this PR fixes the problem?

* Force core-js to version 3 to avoid conflicts between recomponents / gridsome / nuxt
* Disable symlink resolve on webpack level in demo project
* Restore dependencies missed by yarn
* Fix issues in tests

### Check lists

- [x] Readme file updated with actual description and example
- [ ] Added all ARIA attributes required for component
- [x] Added tests for both browser and SSR render and for all components properties
- [ ] Added story with all knobs and actions

### Additional Comments (if any)

* This could also work with local version of Recomm or KYC via symlinks (npm / yarn)
* After this we can try to include build with nuxt / gridsome demo in CI in parallel with main build.
* https://cli.vuejs.org/guide/troubleshooting.html#symbolic-links-in-node-modules

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,33 +17,28 @@ notifications:
   email: true
 node_js:
   - '12'
+before_install:
+  - export TZ=Asia/Tokyo
```
Comment (followyouhome) [inline .travis.yml]: Temporal fix for RDateRange tests

File: packages/recomponents/src/directives/r-content/r-content.spec.js [hunk 2]
```diff
@@ -2,23 +2,35 @@ import {shallowMount} from '@vue/test-utils';
...
+        </div>
+    `,
 };
 
 const wrapper = shallowMount(mockTemplate);
 
 describe('r-content.js', () => {
     it('Should render the correct HTML structure', () => {
-        expect(wrapper.html()).toBe('<div><div class="r-component-content"><section class="r-component-content-is-primary"><p>Text</p></section> <section class="r-component-content-is-secondary"><p>Text</p></section></div></div>');
+        expect(wrapper.html().replace(/[\n\s]/g, '')).toBe(`
```
[the remark this answers is not in the document]
Comment (followyouhome) [reply packages/recomponents/src/directives/r-content/r-content.spec.js]: Bellissimo, missed this feature.

File: packages/recomponents/src/components/r-date-range/__snapshots__/r-date-range.spec.js.snap [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
-exports[`r-date-range.vue renders props when passed 1`] = `<r-popper-stub esctohide="true" autohide="true" globalautohide="true" offset="4" direction="horizontal" position="bottomEnd" slidefrom="fade" duration="0.2" class="r-date-range"></r-popper-stub>`;
-
-exports[`r-date-range.vue should render via SSR and match snapshot 1`] = `
+exports[`r-date-range.vue renders props when passed 1`] = `
 <div class="r-popper r-date-range">
   <div role="group" class="r-button-group"><button class="r-button r-button-type-default r-button-size-regular r-has-icon-left"><svg class="r-icon r-icon-20 r-date-range-calendar-icon">
         <use xlink:href="#icon-calendar" xmlns:xlink="http://www.w3.org/1999/xlink"></use>
       </svg> <span title="America/Montreal" class="r-date-range-label">
-                    Jan 1, 2020 - May 1, 2020
+                    Jan 1, 2020 - Jan 2, 2020
```
[the remark this answers is not in the document]
Comment (followyouhome) [reply packages/recomponents/src/components/r-date-range/__snapshots__/r-date-range.spec.js.snap]: But anyway snapshot is not useless - just to point your eyes that something was changed

File: .travis.yml (hunk 1 above)
Comment (agualis) [reply .travis.yml]: Is this affecting our unit tests? Or is it related with anything else?

Comment (followyouhome) [reply .travis.yml]: Only unit tests - I think our timezone mock require updates

File: README.md [hunk 4]
```diff
@@ -24,6 +24,20 @@ As the number of Web Apps developed at Rebilly grows larger, it has become appar
...
+* `yarn dev`
+
+Or run this below from project root:
+
+* `yarn workspace @rebilly/recomponents dev`
+
+In developer mode each update of component trigger rebuild of Recomponents and this update should be catched by end application to trigger one more rebuild. In result you can check update in your application immideatly.
+
+With workspaces there is no need to install dependencies inside of each folder - single run of `yarn` inside of project root is enough.
+
```
Comment (agualis) [inline README.md]:
```suggestion

We use [Yarn Workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) to make development easier and efficient. Workspaces allow developers to check how the latest **local build** of Recomponents work within an application created with Gridsome or Nuxt **without needing to release that version**.
Running your application (Gridsome/Nuxt/others) against a local version of recomponents allows manually detecting issues that are not covered by unit tests with SSR support before releasing. 
If you would like to setup a similar workflow with your own project, [Yarn Link](https://classic.yarnpkg.com/en/docs/cli/link/) could provide same functionality.

### Development workflow

When you run Recomponents in developer mode, any update in Recomponents will automatically trigger the rebuild of Recomponents and that also will trigger the rebuild of the end client application. In result, **you can check local Recompomnent's updates in your application immediately**. 

### Enabling workspaces

First step is running yarn from `project's root` to enable workspaces

* `yarn dev`

Then, with workspaces you can choose running any npm script from inside of `packages/recomponents` or from the `project's root`.

#### Running Recomponents in developer mode from `/packages/recomponents`

* `yarn dev`

#### Running Recomponents in developer mode from `project's root`:

* `yarn workspace @rebilly/recomponents dev`

With workspaces there is no need to install dependencies inside of each folder - single run of `yarn` inside of project root is enough.

```

Comment (followyouhome) [reply README.md]: Great update, thanks a lot 👍

**Pull request merged**